moonscript/base.moon

```diff
@@ -16,15 +16,15 @@ create_moonpath = (package_path) ->
   moonpaths = for path in *split package_path, ";"
     prefix = path\match "^(.-)%.lua$"
     continue unless prefix
-    prefix .. ".moon"
+    prefix .. ".moon;"..prefix .. ".litmoon"
   concat moonpaths, ";"
 
-to_lua = (text, options={}) ->
+to_lua = (text, options={}, litmoon=false) ->
```

Comment:
I don't think adding another argument specific to litmoon is a good change to this interface. (If we add more formats are we supposed to keep adding new arguments?) Any reason you didn't re-purpose `options`?